### ISSUE TITLE
Confusion about training

### ISSUE THREAD

**Issue by HalvesChen** [MIV-XJTU/FSDrive#25]
Thanks for your great work!
I have a question about your model training. What was the final loss your model converged to? I also observe that the number of epochs seems quite large—I am wondering if you're seeing any signs of overfitting as a result.

**Comment by missTL** [comment]
If I remember correctly, the loss finally converged around 0.1. Since we couldn't effectively know the performance on the test set during training using Llama Factory, we didn't adjust the epoch very many times. You can conduct multiple experiments to select the best hyperparameters.

**Comment by HalvesChen** [comment]
Thank you for your reply~ 
Could you possibly provide the training details as much as possible? For example, the grad norm curves, accuracy on the training set, etc. 
Also, I would like to confirm whether FSDrive has used ViT to further extract semantic features from the generated front view?

**Comment by HalvesChen** [comment]
Also, when you mentioned converging to 0.1, is that for pretraining or SFT? Could you please tell me roughly what each stage converges to?

**Comment by missTL** [comment]
We did not further extract features from the generated front view. Because the model can generate this front view, it proves that the model already knows the future scene. Further extracting features from the generated front view might be a direction worth exploring.
It may not be important to overly focus on what the final value of the loss is, as long as the loss decreases and converges.
